test/fixtures/mixed-global-scoped.out.js

```diff
@@ -1,8 +1,8 @@
 import _JSXStyle from 'styled-jsx/style';
-const Test = () => <_JSXStyle styleId={188072295} css={"p { color: red }"} />;
+const Test = () => <_JSXStyle styleId={188072295} css={" p {color: red }"} />;
 
 export default (() => <div data-jsx={793889750}>
     <p data-jsx={793889750}>test</p>
-    <_JSXStyle styleId={3149549172} css={"body { background: red }"} />
+    <_JSXStyle styleId={3149549172} css={" body {background: red }"} />
```

Comment:
I think that the space is there because of this https://github.com/thysultan/stylis.js/blob/master/stylis.js#L905